code/lib/preview-api/src/modules/store/csf/composeConfigs.ts

```diff
@@ -55,7 +59,8 @@ export function composeConfigs<TRenderer extends Renderer>(
       ...allArgTypeEnhancers.filter((e) => !e.secondPass),
       ...allArgTypeEnhancers.filter((e) => e.secondPass),
     ],
-    globals: getObjectField(moduleExportList, 'globals'),
+    globals: initialGlobals, // deprecated
```

Comment:
What is this used for? This seems confusing/wrong?

Reply:
Technically speaking this is a breaking change to portable stories if we remove it.

Reply:
Why?

Reply:
If some user code calls `composeConfigs` and expects `globals` to be set, and suddenly it is not set, that's a breaking change

Reply:
OK